src/transaction-hooks.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('debug')('feathers-knex-transaction');
```

Comment:
Alternatively we could also export it as `export default { transaction: { start, end, rollback } }` and use it as `hook.transaction.start` etc.